.gitattributes

```diff
@@ -9,4 +9,9 @@
 # https://stackoverflow.com/a/51564689 and
 # https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
 .yarn/** binary
-public/images/** binary
+
+# Exclude vendored files from GitHub statistics. Note that this also includes
+# legacy v2 question client files, which is first-party code that we intend
+# to never touch again.
+apps/prairielearn/public/javascripts/** linguist-vendored
+apps/prairielearn/public/localscripts/** linguist-vendored
```

Comment:
An argument could be made that we should only ignore specifically the files that are vendored (that is, don't ignore `MCQClient.js`, `MTFClient.js`, etc). I'm open to that, but I lean towards pretending that v2 questions don't exist.

Reply:
I'm very much on board with ignoring v2 questions!